lib/cucumber/listener/pretty_formatter.js

```diff
@@ -16,9 +17,9 @@ function PrettyFormatter(options) {
   };
 
   var characters = {};
-  characters[Cucumber.Status.AMBIGUOUS] = '✖';
-  characters[Cucumber.Status.FAILED] = '✖';
-  characters[Cucumber.Status.PASSED] = '✓';
+  characters[Cucumber.Status.AMBIGUOUS] = figures.warning;
```

Comment:
Please keep this as `figures.cross`. Failed and ambiguous are both treated as "Failures" in the summary output so I'd like them to use the same symbol